src/components/course/enrollment/components/ToCoursewarePage.jsx

```diff
@@ -17,7 +17,7 @@ const ToCoursewarePage = ({
     subscriptionLicense,
     enrollmentUrl,
   });
-  const landingUrl = shouldUseEnrollmentUrl ? enrollmentUrl : userEnrollment.courseRunUrl;
+  const landingUrl = shouldUseEnrollmentUrl ? enrollmentUrl : userEnrollment.linkToCourse;
```

Comment:
This is the fix for the "View course" CTA specifically. See other comment for more details about `linkToCourse`.